.travis.yml

```diff
@@ -13,4 +13,4 @@ before_script:
   - mysql -u root < database/setup.sql
   - mysql -u ruddweb_test --password=public < database/reset.sql
 script:
-  - py.test --cache-clear
+  - python -m unittest
```

Comment:
@dkong1796, what's the rationale for using `unittest` instead of `py.test`?
I'm running into issues with pytest's assert.